### DISCUSSION TITLE
수상자 및 수상팀 발표

### DISCUSSION THREAD

**Discussion by justinyoo** [devrel-kr/HackaLearn#427] [0 공지사항]
그동안 HackaLearn 이벤트에 참여해 주셔서 감사 드립니다!

아래는 HackaLearn 챌린지별 수상자 혹은 수상팀입니다. 축하드립니다!

* 클라우드 스킬 챌린지 완주자 👉 [수상자 보러 가기](https://github.com/devrel-kr/HackaLearn/blob/main/CLOUD_SKILL_CHALLENGES.md)
* SNS 인증샷 챌린지 &ndash; 두 명 👉 [김시연](https://www.instagram.com/p/CSlyUPzpfWr/?utm_medium=copy_link), [박주은](https://www.instagram.com/p/CSlGgG8Ag0a/?utm_medium=copy_link)
* 블로그 후기 챌린지 &ndash; 세 명 👉 [김호정](https://velog.io/@hokim/HackaLearn-%EC%B0%B8%EC%97%AC-%ED%9B%84%EA%B8%B0-%ED%8C%80%EC%98%A4%EB%AF%80-%EB%9D%BC%EC%9D%B4%EC%8A%A4), [김예림](https://blog.naver.com/sedagrq821/222470813632), [박수빈](https://subeen.io/blog/devs/2021-08-15-HackaLearn/)
* 가장 창의적인 애플리케이션 &ndash; 한 팀 👉 [KING](https://github.com/devrel-kr/HackaLearn/blob/main/teams/KING.md)
* 가장 UX가 뛰어난 애플리케이션 &ndash; 한 팀 👉 [생생한국통](https://github.com/devrel-kr/HackaLearn/blob/main/teams/생생한국통.md)
* 가장 사회 공헌적인 애플리케이션 &ndash; 한 팀 👉 [월급두배받는법](https://github.com/devrel-kr/HackaLearn/blob/main/teams/월급두배받는법.md)

**Comment by subeenpark-io** [comment]
감사합니다! 행사 너무 열정적으로 진행해주셔서 덩달아 즐겁게 참여했습니다 :)

**Reply by justinyoo** [reply]
수상을 축하드립니다!

**Reply by jung-youjin** [reply]
수상 축하드립니다!🥳

**Comment by ksiyeon27** [comment]
안녕하세요! 행사 너무 즐거웠습니다:) 
sns 챌린지 수상자인데 메일이 오지 않아 혹여 잘못 등록되었나 싶어서 메일 주소를 다시 남깁니다! siyeonk27@gmail.com 으로 안내문을 다시 보내주실 수 있을까요?

**Reply by justinyoo** [reply]
@ksiyeon27 스팸메일 박스를 확인해 보시겠어요? 어젯밤 1시 8분 경에 성공적으로 보내졌습니다.

![image](https://user-images.githubusercontent.com/1538528/130149189-f9577b36-71d2-4b76-85d8-bab49fb39980.png)

**Comment by jung-youjin** [comment]
수상자분들, 참가자분들 다 너무 고생 많으셨습니다!! 그리고 너무 감사합니다!! 😚🙏🏼

**Comment by JaeSeoKim** [comment]
안녕하세요! Hackalearn 행사 재미있게 잘 즐겁고 유익했습니다!
다름이 아니라 이번 UX 수상자인데 연락하는 경로에 대해 문의를 하고자 남깁니다.
해당 이메일에 답장을 주시라고 적어주셨는데 아래에 있는 이메일 주소는 보내기 전용이라고 적어져 있어서 어디로 연락을 해야 할지 궁금합니다.

**Reply by justinyoo** [reply]
그리로 답장 주세요.

**Reply by justinyoo** [reply]
다시 한 번 수상 축하 드립니다!
